python/ray/rllib/agents/ddpg/ddpg.py

```diff
@@ -16,6 +16,18 @@
 # yapf: disable
 # __sphinx_doc_begin__
 DEFAULT_CONFIG = with_common_config({
+    # === Twin Delayed DDPG (TD3) and Soft Actor-Critic (SAC) tricks ===
+    # twin Q-net
+    "twin_q": False,
+    # delayed policy update
+    "policy_delay": 1,
+    # target policy smoothing
+    "use_gaussian_noise": False,
```

Comment:
Call this `smooth_target_policy`? You can then add in the comment "this also forces the use of gaussian instead of OU noise for exploration".